client/src/components/ForgotPassword/CheckEmail/CheckEmailContainer.tsx

```diff
@@ -1,79 +1,40 @@
-import { renderField } from "../../commonComponents/reduxFormFields";
-import { checkEmail } from "../../../store/checkEmail/actions";
-import validate from '../../../utils/validation'
-import "./../../styles/style.scss";
-import logo from "./../../assets/logo.png";
+import React, { ReactNode } from 'react';
+import { reduxForm } from 'redux-form';
+import { connect } from 'react-redux';
+import { checkEmail } from '../../../store/checkEmail/actions';
+import validate from '../../../utils/validation';
+import './../../styles/style.scss';
+import CheckEmail from './CheckEmail';
+import { FormProps } from 'reactstrap';
 
 class CheckEmailContainer extends React.Component<any> {
-  onSubmit = (email: string) => {
-    return this.props.checkEmail(email);
-  };
-  render() {
-    const { handleSubmit, submitting } = this.props;
-    return (
-      <div className="container-fluid header">
-        <div className="row justify-content-center align-items-center">
-          <div className="col-sm-8 col-md-6 col-xl-5">
-            <Form className="mt-4 bg-white" onSubmit={handleSubmit(this.onSubmit)}>
-              <div className="border">
-                <FormGroup className="col-lg-10 offset-lg-1 text-center">
-                  <Link to="/">
-                    <img className="picture" src={logo} alt="logo" />
-                  </Link>
-                  <Field
-                    name="email"
-                    type="text"
-                    component={renderField}
-                    label="email"
-                    className="form-control form-control-lg mt-3"
-                    placeholder="E-mail"
-                  />
-                  <Button
-                    className="mt-3"
-                    disabled={submitting}
-                    color="danger"
-                    size="lg"
-                    block
-                  >
-                    Send
-                  </Button>
-                  {submitting && <Spinner className="mt-3" color="dark" />}
-                </FormGroup>
-                <div className="or-devider">
-                  <span />
-                  OR
-                  <span />
-                </div>
-                <FormGroup className="text-center login_google">
-                  <p className="pt-2">
-                    Still don't have an account?
-                    <Link to="/register" className="pl-1 text-danger">
-                      Register
-                    </Link>
-                  </p>
-                </FormGroup>
-              </div>
-            </Form>
-          </div>
-        </div>
-      </div>
-    );
-  }
+    constructor(props: any) {
+        super(props);
+        this.onSubmit = this.onSubmit.bind(this);
+    }
+
+    public onSubmit(email: string): any {
+        return this.props.checkEmail(email);
+    }
+
+    public render(): ReactNode {
+        const {handleSubmit, submitting}: FormProps = this.props;
+        return (
+            <CheckEmail
```

Comment:
I believe we agreed not to move CheckEmail in separate component as there is no reason for it